terraform/backends/ab2d-dev.s3.tfbackend

```diff
@@ -0,0 +1,6 @@
+bucket         = "ab2d-dev-tfstate"
+key            = "ab2d-dev/terraform/api/terraform.tfstate"
+region         = "us-east-1"
+dynamodb_table = "ab2d-dev-api-tfstate-table"
```

Comment:
The table does not need to match the key - many keys can be kept in each dynamo table. It's silly that AB2D terraform is set up to user a different table for each key in AB2D. So this should be changed as follows:

```suggestion
dynamodb_table = "ab2d-dev-tfstate"
```

Reply:
As discussed in Slack, we'll need to create this table before applying terraform